apps/aehttp/src/sc_ws_handler.erl

```diff
@@ -272,6 +272,7 @@ read_channel_options(Params) ->
          Read(<<"responder_amount">>, responder_amount, #{type => integer}),
          Read(<<"channel_reserve">>, channel_reserve, #{type => integer}),
          Read(<<"ttl">>, ttl, #{type => integer, mandatory => false}),
+         Read(<<"protocol_vsn">>, protocol_vsn, #{type => integer, mandatory => false}),
```

Comment:
@uwiger Could you please clarify how this relates to [this doc re channel params](https://github.com/aeternity/protocol/blob/3a00b538ed47efa6ab7bb205f07456e06134ff44/node/api/channels_api_usage.md#channel-parameters)? Any docs requiring update?